### DISCUSSION TITLE
Weblate

### DISCUSSION THREAD

**Discussion by Mendiak** [elementary/triage#499] [Q&A] [answered]
Hi, a newbie here trying to contribute to the project. 

I tried to add spanish translations at Weblate but i see a lot of old suggesstions that have not aproved/reviewed. Is it abandoned?

**Comment by ryonakano** [comment, marked as answer]
I guess it's because there are no active translator who have permission to save Spanish translation. If you're interested in translating elementary OS, refer to [this page](https://docs.elementary.io/contributor-guide/localization/translations).
